<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartDesignGui::TaskFilletParameters</class>
 <widget class="QWidget" name="PartDesignGui::TaskFilletParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="message">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonRefAdd">
       <property name="toolTip">
        <string>Click the button to enter toggle mode. Click
again to return to normal mode.

In toggle mode, you can select geometry either
in the list below, or in the 3D view to toggle
its state.

In normal mode, you can only add new geometry
by selection. Existing geometry references can
be removed using the Delete key.</string>
       </property>
       <property name="text">
        <string>Toggle</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetReferences">
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Radius:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Gui::QuantitySpinBox" name="filletRadius">
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
